src/components/ClearButton.jsx

```diff
@@ -3,21 +3,18 @@ import { Text, StyleSheet, View } from "react-native";
 
 export const ClearButton = () => {
   return (
-    <View style = {styles.container}>  
-      <Text style={styles.clear}>
-        CLEAR ALL DONE
-      </Text>
+    <View style={styles.container}>
+      <Text style={styles.clear}>CLEAR ALL DONE</Text>
```

Comment:
What do you think of making this `Text` a `Pressable` now? Maybe we can do it in a different branch to make it clearer 💪